kernel/build/kernel-config-deb.sh

```diff
@@ -5939,7 +5939,7 @@ CONFIG_DEBUG_BUGVERBOSE=y
 CONFIG_DEBUG_INFO=y
```

Comment:
How about this instead to avoid pinning the DWARF version?
```suggestion
CONFIG_DEBUG_INFO=y
CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT=y
```

Reply:
(This is just the addition of `CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT=y`, `CONFIG_DEBUG_INFO=y` isn't touched -- it shows up this way due to how Github renders "Suggested changes".)

Reply:
Sure, sounds good.